src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,614 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures the performance of aggregation expressions that
+  can take in a variable number of expressions as arguments.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE variadic aggregation expressions and numbers on the 'standalone'
+  environment for the classic variadic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 32
+  ListOfTenInts: &ListOfTenInts {^Choose: {from: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"], deterministic: 1}}
+  ListOfFiftyInts: &ListOfFiftyInts {^Choose: {from: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10",
```

Comment:
I guess it's not possible to use ^Inc and get a string, but: the lists of 50 and 100 integers ought to be simplified further . Could this be done by joining two deterministic choose lists; something like ["", "1", "2", "3", "4"] and the list of 10, then ["", "1", "2", "3", "4",  "5", "6", "7", "8", "9"]  and the list of 10? This would make it so that we start at 0, but this is definitely worth it if we can avoid hard coding a list.